README.md

```diff
@@ -12,206 +12,614 @@ A command-line tool for managing TEE deployments on Phala Network, from local de
+
+**Options:**
+- `-i, --image <image>`: Image name (required)
+- `-t, --tag <tag>`: Image tag (required)
+- `-f, --file <file>`: Path to Dockerfile (defaults to 'Dockerfile')
+
+**Example:**
+```bash
+phala docker build --image my-tee-app --tag v1.0.0 --file ./Dockerfile
+```
+
+#### Push Docker Image
+
+```bash
+phala docker push [options]
+```
+
+Push a Docker image to Docker Hub.
+
+**Options:**
+- `-i, --image <image>`: Image name (required)
+- `-t, --tag <tag>`: Image tag (required)
+
+**Example:**
+```bash
+phala docker push --image my-tee-app --tag v1.0.0
+```
+
+#### List Docker Image Tags
+
+```bash
+phala docker tags [options]
+```
+
+List all tags for a Docker image on Docker Hub.
+
+**Options:**
+- `-i, --image <image>`: Image name (required)
+- `-j, --json`: Output in JSON format
+
+**Example:**
+```bash
+phala docker tags --image my-tee-app
+```
+
+#### Build Docker Compose File
+
+```bash
+phala docker build-compose [options]
+```
+
+Build a Docker Compose file for your TEE application.
+
+**Options:**
+- `-i, --image <image>`: Image name (required)
+- `-t, --tag <tag>`: Image tag (required)
+- `-u, --username <username>`: Docker Hub username
+- `-e, --env-file <envFile>`: Path to environment file
+- `-v, --version <version>`: Template version to use (basic, eliza-v1, eliza-v2)
+
+**Example:**
+```bash
+phala docker build-compose --image my-tee-app --tag v1.0.0 --env-file ./.env
+```
+
+#### Run Local Docker Compose
+
+```bash
+phala docker run-local [options]
+```
+
+Run a Docker Compose file locally for testing.
+
+**Options:**
+- `-c, --compose <compose>`: Path to Docker Compose file
+- `-e, --env-file <envFile>`: Path to environment file
+
+**Example:**
+```bash
+phala docker run-local --compose ./tee-compose.yaml --env-file ./.env
+```
+
+### TEE Simulator Commands
+
+Commands for managing the local TEE simulator for development and testing.
+
+#### Start Simulator
+
+```bash
+phala simulator start [options]
+```
+
+Start the TEE simulator locally for development and testing.
+
+**Options:**
+- `-i, --image <image>`: Simulator image (defaults to 'phalanetwork/phala-pruntime:latest')
+
+**Example:**
+```bash
+phala simulator start
+```
+
+#### Stop Simulator
+
+```bash
+phala simulator stop
+```
+
+Stop the running TEE simulator.
+
+**Example:**
+```bash
+phala simulator stop
+```
+
+### Configuration Commands
+
+Commands for managing CLI configuration settings.
+
+#### Get Configuration Value
+
+```bash
+phala config get <key>
+```
+
+Get a specific configuration value.
+
+**Arguments:**
+- `key`: Configuration key to retrieve
+
+**Example:**
+```bash
+phala config get apiUrl
+```
+
+#### Set Configuration Value
+
+```bash
+phala config set <key> <value>
+```
+
+Set a configuration value.
+
+**Arguments:**
+- `key`: Configuration key to set
+- `value`: Value to set (can be a string, number, boolean, or JSON)
+
+**Example:**
+```bash
+phala config set defaultVcpu 2
+phala config set apiUrl "https://custom-api.phala.cloud"
```

Comment:
Not sure if you have switched to the latest version of cloud API, which has teepod availability check and switch function

Reply:
I added the check for teepod availability for creating a CVM.